src/client/app/appReducer.js

```diff
@@ -117,6 +119,17 @@ export default (state = initialState, action) => {
         },
       };
     }
+    case appTypes.SHOW_POST_MODAL:
+      return {
```

Comment:
@Sekhmet I don't see your comment anymore, but I do remember you suggesting maybe move this in `postsReducer`.

I don't think `postsReducer` would be a good place, `postsReducer` feels like a place to hold all post data.
While `appReducer` is meant to hold data for the entire app as a whole, since you are technically modifying the app as whole here, by introducing a modal into the DOM, I figured that the `appReducer` would be the best place. Let me know what you think, but these are my thoughts on it. I have no preference though

Reply:
Alright, we can store this in appReducer.